### PR TITLE
Add DBeaver guide

### LINKED ISSUE #1086: Change required: How to use dbeaver to connect rasingwave
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

Docker deploys rasingwave, how to use dbeaver to connect rasingwave, how to set the environment, such as database name, user name, password, etc., is it the same as postgre? I tried it, but I couldn’t connect.
image
image
image
image

### Reference

_No response_

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  Add DBeaver guide. Not included in sidebar in anticipation of restructuring.

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1086
https://github.com/risingwavelabs/risingwave-docs/issues/1073

## For reviewers

- **Preview**

https://pr-1213.d2fbku9n2b6wde.amplifyapp.com/docs/upcoming/dbeaver-integration/

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Comment by neverchanje** [comment]
Hi Emily, the content is good. But I couldn't find this page in the preview site: 

https://pr-1213.d2fbku9n2b6wde.amplifyapp.com/docs/current/superset-integration/

I think this page should be placed in the visualization section. 

Screenshot 2023-08-28 at 18 53 45

And the integration page should also link the dbeaver support to the new guide.

Screenshot 2023-08-28 at 18 54 30

**Review by neverchanje — APPROVED** [review]
LGTM once above issues are resolved.

**Comment by hengm3467** [comment]
I'll be adjusting the sidebar later today.

File: docs/guides/dbeaver-integration.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - Password: The password associated with the provided username. By default, there is no password for `root`.
+
+    <img
+    src={require('../images/dbeaver-connect-pg.png').default}
+    alt="Fill in connection settings in DBeaver"
+    />
+
+4. Click **Test connection**. If there are no errors, click **Finish**.
+
+Now that the connection is established, the RisingWave database should be listed in the DBeaver sidebar under **Database navigator**. By default, anything created in RisingWave, such as tables, sources, or materialized views, will be listed under the `dev` database and the `public` schema.
```
Comment (hengm3467) [inline docs/guides/dbeaver-integration.md]: Ideally, we can test the connection with some simple queries like creating a table and insert some values, and then query the table (maybe having a screenshot). Overall looks great. Thanks!

**Review by hengm3467 — APPROVED** [review]

**Pull request merged**